cloudtests.py

```diff
@@ -62,16 +62,34 @@ def test_net_name(self):
 class TestJournalWritten(unittest.TestCase):
 
     def test_journal_written(self):
-        """Test to check if journal gets written to disk"""
+        """
+        Test to check if journal gets written to disk
+        And make sure that journal logs get written to /var
+        """
 
+        # Find PID
         out, err, eid = system("systemctl show systemd-journald.service -p MainPID")
         out = out.decode('utf-8')
         pid = out[8:-1]
+
+        # Find UID
+        out, err, eid = system("whoami")
+        username = out.decode('utf-8')
+        out, err, eid = system("id -u {0}".format(username))
+        uid = out.decode('utf-8').strip('\n')
+
+        # Journal log
         out, err, eid = system("sudo ls -l /proc/{0}/fd/ | grep journal".format(pid))
```

Comment:
Yes we need to have this to find out `journal` from the output of `sudo ls -l /proc/pid/fd/` just to avoid showing the huge output in case of error.